index.html

```diff
@@ -36,5 +36,135 @@ <h3>LET'S CONNECT</h3>
                 <a href="#"><li><img src="./Vector (2).png" alt="victory"></li></a>
             </ul>
     </section>
+<section class="section-2">   
+<div class="grid-container1">
+       <div class="grid-item1"><img src="Snapshoot Portfolio.png" alt="Snapshoot portfolio"></div>
+       <div class="grid-item2">
+        <h2>Tonic</h2>
+        <ul>
+            <li style="list-style: none;" id="canopy">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p>A daily selection of privately<br>
+             personalized reads; no accounts or<br>
+              sign-ups required.</p>
+              <ul style="list-style: none;" class="language-button">
+                <li><button type="submit" class="html-button">html</button></li> 
+                <li><button type="submit" class="css-button">css</button></li>
+              <li><button type="submit" class="javascript-button">javascript</button></li>
```

Comment:
[OPTIONAL] 
- [ ] It's better if you make the badges as `span` or something like this instead of `button` since they are not buttons 👍